zips/draft-ecc-zbloc.md

```diff
@@ -0,0 +1,268 @@
+
+Each Constituency initially receives three Voting Units. This MAY be changed by a Governance Proposal.
+
+Constituencies other than the Coin-holder Constituency above are called Organizational Constituencies. The Organizational Constituencies will together elect a representative responsible for establishing a clear process and mechanism for gathering coin-holder sentiment, which will then be used as the decision-making process of the Coin-holder Constituency.
+
+TODO: as Organizational Constituencies make commitments about their decision-making policies, link to those commitments here.
+
+Initial zBloc members MUST take on the responsibility to encourage, nurture, and welcome potential new Constituencies.
+
+**Rationale:** Relentless decentralization is zBloc’s core value and mission. It is the unifying ethos that will keep zBloc’s doors open to new contributors. This continuous expansion is in the best interests of the growing Zcash community. 
+
+## Voting
+
+A suitable voting mechanism is needed that —among other design requirements— makes the votes cast by each Constituency (i.e. the number of Voting Units cast for Approval and for Rejection) publically visible. A possible design that meets this requirement is suggested in [^draft-ecc-onchain-accountable-voting] ("On-chain Accountable Voting").
+
+Each independent zBloc Constituency decides how to cast its Voting Units for each Governance or Funding Proposal, provided it is eligible to vote on that proposal (see the next section). If its decision-making policy permits, it MAY cast fewer than its total number of Voting Units, and MAY cast some Voting Units for Approval and some for Rejection. This might be used to publically acknowledge internal dispute over a decision, for example.
+
+### Conflicts of interest
+
+If a Constituency has a conflict of interest for a given decision, such as potentially being the recipient of funds or additional authority, then its Voting Units are not eligible to be cast for that vote. This affects the [Approval Threshold] discussed below, which is calculated as a proportion of the remaining Eligible Voting Units.
+
+A Constituency is not automatically considered to have a conflict of interest merely because the decision affects it — for example, when changing its decision-making policy, or because it would need to make changes to software if the Proposal were Approved.
+
+### Approval Thresholds
+
+The Approval Threshold is the absolute majority, as a proportion of the total number of Eligible Voting Units, required for a Proposal to be Approved. As stated earlier, Eligible Voting Units do not include those of Constituencies that have a conflict of interest for this Proposal. However, they do include the Voting Units of Constituencies that have no conflict of interest but choose to abstain.
+
+By default, a Governance Proposal requires a three-fifths Approval Threshold, and MUST have a Deadline at least two weeks from when the Proposal is published on-chain.
```

Comment:
This seems out-of-order; we don't yet have any information about what it means to publish a Proposal on-chain, or who is responsible for that process.

Reply:
It was hard to find a sensible order that didn't involve forward-references. There might be a better order, but in any case remaining forward-references should be explicit links; feel free to fix that.